stormgls/stormgls.py

```diff
@@ -25,6 +27,8 @@
 
 LSSOURCE = "stormgls"
 
+CWD = pathlib.Path(__file__).parent.absolute()
```

Comment:
I know nvim has the concept of stashing things in "~/.local/share/nvim". I wonder if that's something we should also do to be good citizens.

Reply:
If it helps start the search, this looks like how telescope does it: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/config.lua#L523